messaging/src/main/java/org/springframework/security/messaging/access/expression/MessageExpressionConfigAttribute.java

```diff
@@ -53,4 +71,15 @@ public String getAttribute() {
 	public String toString() {
 		return authorizeExpression.getExpressionString();
 	}
+
+	@Override
+	public EvaluationContext postProcess(EvaluationContext ctx, Message<?> message) {
+		if (matcher instanceof SimpDestinationMessageMatcher) {
```

Comment:
We either need to validate when this is constructed that matcher is not null or perform a null check here.

Reply:
@rwinch, already validated in the constructor as the previous comment says, anyway the "null instanceof SomeClass" expression evaluates to false.